python/nano/src/bigdl/nano/utils/inference/common/utils.py

```diff
@@ -63,6 +63,9 @@ def throughput_calculate_helper(iterrun, baseline_time, func, *args):
     '''
     A simple helper to calculate average latency
     '''
+    # test run two times for more accurate latency
+    for i in range(2):
+        func(*args)
```

Comment:
so if I understand this correct, this should be a warm up?

Reply:
yes, it's a warm up.